src/shop-checkout.html

```diff
@@ -120,7 +120,7 @@
           <iron-form id="checkoutForm"
               on-iron-form-response="_didReceiveResponse"
               on-iron-form-presubmit="_willSendRequest">
-            <form method="post" action="/data/sample_success_response.json">
+            <form method="post" action="data/sample_success_response.json">
```

Comment:
This is an interesting example. IRL, this API probably won't be static, and probably will live at a fixed location even though the client features differential serving. If I naively follow a "best practice" of never using absolute URLs for things (so as to take advantage of differential serving), it may be difficult to reason about the correctness of changes like this (which, again IRL, could actually be a bug). The base, in this case, creates a dissonance between the URLs for static resources and non-static ones (unless my server already fixes this for me in a magical way).